glyph/src/ttfp.rs

```diff
@@ -16,6 +16,44 @@ impl From<GlyphId> for owned_ttf_parser::GlyphId {
     }
 }
 
+/// A pre-rendered image of a glyph, usually used for emojis or other glyphs
+/// that can't be represented only using an outline.
+#[derive(Debug, Clone)]
+pub struct GlyphImage<'a> {
+    /// offset of the image from the normal origin (top at the baseline plus
+    /// ascent), measured in pixels at the image's current scale
+    pub origin: Point,
+    /// current scale of the image in pixels per em
+    pub scale: f32,
+    /// raw image data (not a bitmap)
+    pub data: &'a [u8],
+    /// format of the raw data
+    pub format: GlyphImageFormat,
+}
+
+impl<'a> From<owned_ttf_parser::RasterGlyphImage<'a>> for GlyphImage<'a> {
+    fn from(img: owned_ttf_parser::RasterGlyphImage<'a>) -> Self {
+        GlyphImage {
+            origin: point(img.x.into(), img.y.into()),
+            scale: img.pixels_per_em.into(),
+            data: img.data,
+            format: match img.format {
+                owned_ttf_parser::RasterImageFormat::PNG => GlyphImageFormat::PNG,
+            },
+        }
+    }
+}
+
+/// Valid formats for a [GlyphImage]
+///
+/// Note: currently only PNG images are actually returned by glyph_raster_image.
+// Possible future formats:  SVG, JPEG, TIFF
+#[non_exhaustive]
+#[derive(Debug, Clone)]
+pub enum GlyphImageFormat {
+    PNG,
```

Comment:
```suggestion
                owned_ttf_parser::RasterImageFormat::PNG => GlyphImageFormat::Png,
            },
        }
    }
}

/// Valid formats for a [GlyphImage]
///
/// Note: currently only PNG images are actually returned by glyph_raster_image.
// Possible future formats:  SVG, JPEG, TIFF
#[non_exhaustive]
#[derive(Debug, Clone)]
pub enum GlyphImageFormat {
    Png,
```

Reply:
I think the enum value should be PortableNetworkGraphics if it has to be camel case, but that's mostly a personal opinion.

Reply:
No one calls it that though. Capitals vs pascal case is a convention thing that changes lang to lang, but in rust the convention is `Png` here. I'm not sure why _ttf-parser_ did it "wrong" but I don't have to repeat it in my own crates (with the exception of _owned-ttf-parser_).

Reply:
But this is minor I can fix this up after merging